ccla-src/cloudclapp-guide/index.adoc

```diff
@@ -52,6 +52,13 @@ The options are "Hourly", "Monthly" and Annual
 
 
 ==== Environment List
+=== Blueprints
```

Comment:
It's very good description but it's more like for "User Manual" (MSA-12129)

For "Screen Description", we should mention "you can do xxx by clicking yyy"
In your case, clicking the card shows the preview, and clicking "Use as A New Environment" creates and environment, etc..
eg. 
```
=== Blueprints
==== Blueprint
You can see ....
==== Use As a New Environment
You can create....
```

Then for "User Manual", we can cover some use-cases which cross over the multiple screens like you mentioned.
Maybe you can transfer some description to https://github.com/ubiqube/docs/pull/110

Reply:
Added all changes in one consolidated PR : https://github.com/ubiqube/docs/pull/110
@hiro-ubiqube Kindly take a look and let us know your thoughts. Closing this PR.